cargo-dist/src/tasks.rs

```diff
@@ -1199,6 +1196,7 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             formula,
             system_dependencies,
             platform_support,
+            bin_aliases: BinaryAliases(package_config.bin_aliases.clone().unwrap_or_default()),
```

Comment:
(style nit) can you compute this earlier up like the rest of the configs?

Reply:
Done